src/server/services/wms/qgswmsparameters.h

```diff
@@ -0,0 +1,240 @@
+/***************************************************************************
```

Comment:
Nice! Just wondering if this restricts in any way the addition of new vendor parameters from the plugins.